hermetic_build/DEVELOPMENT.md

```diff
@@ -160,11 +160,13 @@ python hermetic_build/library_generation/cli/entry_point.py generate \
 1. Run the following command to build the image from source
 
    ```shell
-   docker build \
+   DOCKER_BUILDKIT=1 docker build \
```

Comment:
Can we mention that `DOCKER_BUILDKIT=1` is only needed for local development on CloudTop but not needed in CloudBuild?

Reply:
Added a note below.